crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -210,10 +207,7 @@ impl FromWorld for MeshPipeline {
                 BindGroupLayoutEntry {
                     binding: 3,
                     visibility: ShaderStages::FRAGMENT,
-                    ty: BindingType::Sampler {
-                        comparison: true,
-                        filtering: true,
-                    },
+                    ty: BindingType::Sampler(SamplerBindingType::Comparison),
```

Comment:
Can someone verify if this isn't a behavorial change.

Reply:
+1
Also not sure

Reply:
I dont know what these options should affect. But with other SamplingBindingType::Filtering in panics.
Result looks pretty same.

Reply:
Yeah I think comparison is fine for now. We might be able to rein this in, but this seems like the right choice for the current impl.